cmd/evm/internal/t8ntool/flags.go

```diff
@@ -30,10 +30,13 @@ var (
 		Name:  "trace",
 		Usage: "Output full trace logs to files <txhash>.jsonl",
 	}
-	TraceDisableMemoryFlag = &cli.BoolFlag{
-		Name:  "trace.nomemory",
-		Value: true,
-		Usage: "Disable full memory dump in traces (deprecated)",
+	TraceTracerFlag = &cli.StringFlag{
+		Name:  "trace.tracer",
+		Usage: "The type of tracer. It could be callTracer or prestateTracer",
```

Comment:
Please also update the flag description, as you said we extend the tracer to also support native stuff right?

Reply:
Actually it support native and js tracers, updated.